ui/src/app/components/status-alert-banner-maybe.tsx

```diff
@@ -4,73 +4,74 @@ import { useEffect, useState } from 'react';
 import { StatusAlert, StatusAlertLocation } from 'generated/fetch';
 
 import { statusAlertApi } from 'app/services/swagger-fetch-clients';
-import { firstPartyCookiesEnabled } from 'app/utils/cookies';
+import {
+  isDismissed,
+  saveDismissedMessage,
+} from 'app/utils/dismissed-messages';
```

Comment:
The logic for checking and updating cookies is a bit involved, so it seemed reasonable to have that in utility functions.